plugin/src/main/kotlin/org/gradleweaver/plugins/jlink/JLinkLauncherOptions.kt

```diff
@@ -0,0 +1,43 @@
+package org.gradleweaver.plugins.jlink
+
+import org.gradle.api.Project
+
+class JLinkLauncherOptions(project: Project) {
+
+    private val objectFactory = project.objects
+
+    /**
+     * Runtime options for the JVM.
+     */
+    val vmOptions = objectFactory.listProperty<String>()
+
+    /**
+     * The name of the generated launcher script. If not set, the launcher name will be the name of the
+     * corresponding project.
+     */
+    val launcherName = objectFactory.property(project.name)
```

Comment:
Make the default lazily evaluate `project.name`.

Reply:
Does it matter? I thought the project name will always be defined since it's an immutable property

Reply:
Oh, if it's immutable, then it's fine.